rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/60_knn_search_filter_alias.yml

```diff
@@ -105,3 +105,37 @@ setup:
 
   - match: { hits.hits.1._id: "2" }
   - match: { hits.hits.1.fields.name.0: v1 }
+
+  - do:
```

Comment:
Great idea adding this test, now it is much more obvious.

Reply:
Credit to you for the idea :)